Makefile

```diff
@@ -122,3 +122,16 @@ ci-uregressions:
 
 ci-uregressions-ulong:
 	$(Q)+$(MAKE) -C src uregressions-ulong
+
+# Shortcuts:
+
+1: dune-fstar
+
+2:
+	+$(MAKE) -C src ocaml
+	+$(MAKE) dune-fstar
+
+3:
```

Comment:
FYI @nikswamy @aseemr, this is a way to bootstrap the compiler without verifying/extracting ulib. It won't work if something in ulib that the compiler depends on changed significantly, but it can save you time otherwise